inst/staticexports/package.R

```diff
@@ -3,9 +3,20 @@ is_loaded <- function(x) {
 }
 
 is_installed <- function(package, version = NULL) {
-  installed <- nzchar(system.file(package = package))
+  installed <- requireNamespace(package, quietly = TRUE)
   if (is.null(version)) {
     return(installed)
   }
-  installed && isTRUE(utils::packageVersion(package) >= version)
+  installed && isTRUE(fastPackageVersion(package) >= version)
 }
+
+# Since I/O can be expensive, only utils::packageVersion() if the package isn't already loaded
+fastPackageVersion <- function(pkg) {
```

Comment:
I don't think we need `fast`, since that's assumed to be true of stuff in this package. Also, snake case, so it would be `package_version`.

Reply:
To avoid confusion with `base::package_version()`, which does something different, I went with `get_package_version()`